### ISSUE TITLE
Buzzing on D3/D11

### ISSUE THREAD

**Issue by scoobymooch** [BipeFlyer/Troopduino-Board#2]
Hi!

This project looks absolutely amazing! Very inspiring to see what’s possible with so few components.

I’ve put together your circuit on a breadboard, and am experiencing the most awful buzzing from the Arduino pins (D3/D11).

I’ve tried powering from laptop USB and from batteries (6V). I’ve also removed the DF Player from the circuit. For speakers I’ve tried headphones, and speakers powered by a small 3W amp.

Do you have any advice?

Best,
Matt

**Comment by MissionTrooper-TK-76271** [comment]
Have you tried disconnecting the microphone?  Is the buzzing constant, or just when speaking?  Do you have the bitcrusher settings turned up to high?

**Comment by scoobymooch** [comment]
Hey there. Thanks for the quick reply.

I worked out that the problem is all about the speaker end of things. I stripped it back to just having pin D3 hooked up (with the low pass filter) and through headphones it seems to be working. Then I worked out that I'd been overdriving the input of my little 3W amp, and reducing the input voltage with a resistor divider has fixed that problem.

So good progress today! As soon as I get it working properly and installed in the (not yet finished) helmet I'll be sure to send a photo :)

Thanks again for the quick reply. Apologies for distracting you with my newbie electronics mistakes.

Matt
